common/src/thrift/file_system_master.thrift

```diff
@@ -23,6 +23,7 @@ struct FileInfo {
   18: string userName
   19: string groupName
   20: i32 permission
+  21: string persistenceState
```

Comment:
never mind, we don't need to support backcompat in our thrift RPCs. If we needed backcompat, the thrift enum lets you rename the field and makes it clearer that people need to be careful about changing it.